test/gen-server/migrations.ts

```diff
@@ -116,6 +116,22 @@ describe('migrations', function() {
     // be doing something.
   });
 
+  it('can migrate UserUUID and UserUniqueRefUUID with user in table', async function() {
+    this.timeout(60000);
+    const runner = home.connection.createQueryRunner();
+    for (const migration of migrations) {
+      if (migration === UserUUID) {
+        // Create 400 users to test the chunk (each chunk is 300 users)
+        for (let i = 0; i < 400; i++) {
+          await runner.query(`INSERT INTO users (id, name, is_first_time_user) VALUES (${i}, 'name${i}', true)`);
+        }
+      }
+
+      await (new migration()).up(runner);
+    }
+    await addSeedData(home.connection);
```

Comment:
Thanks for adding a test. One small suggestion: could you add a check here that goes through all the users and checks they each have a unique `ref`? A simple way to check might be to map all of their `ref`s, drop them in a `Set`, and check that you have 400.